src/main/java/org/owasp/esapi/logging/appender/LogPrefixAppender.java

```diff
@@ -44,17 +46,23 @@ public class LogPrefixAppender implements LogAppender {
      * @param logServerIp        Whether or not to record server ip information
      * @param logApplicationName Whether or not to record application name
      * @param appName            Application Name to record.
+     * @param omitEventTypeInLogs Application Name to record.
      */
-    public LogPrefixAppender(boolean logUserInfo, boolean logClientInfo, boolean logServerIp, boolean logApplicationName, String appName) {
+    public LogPrefixAppender(boolean logUserInfo, boolean logClientInfo, boolean logServerIp, boolean logApplicationName, String appName, boolean omitEventTypeInLogs) {
```

Comment:
Ok, I have changed back. I have a question where should I get the OmitEventTypeInLogs property? 

I could create a static variable in Slf4JLogFactory and use the static method to get this property and create a setOmitEventTypeInLogs method in LogPrefixAppender to set the value;
I can do that when the LogPrefixAppender instanced in Slf4JLogFactory static method. So I would get the property and create a setOmitEventTypeInLogs method in LogPrefixAppender to set the value.

Which approach do you think would be better or do you have a suggestion?

Reply:
ESAPI generally makes the assumption (except when running JUnit tests) that the value of a a specific property provided in ESAPI.properties doesn't change in the middle of your application, so as a result, it's generally fine to assume one a property is set, you can store its value in a static class variable.

That said, what I was thinking in the context of LogPrefixAppender was rather than changing the CTOR (or adding a new one), change your declaration of:
```java
    private final boolean omitEventTypeInLogs;
```
to
```java
    import static org.owasp.esapi.PropNames.OMIT_EVENT_TYPE_IN_LOGS;
    ...
    private static boolean omitEventTypeInLogs;

    static {
        try {
            omitEventTypeInLogs =
                ESAPI.securityConfiguration().getBooleanProp( OMIT_EVENT_TYPE_IN_LOGS );
        } catch (Exception ex) {    // Usually this will be ConfigurationException
            omitEventTypeInLogs = false;
        }
    }
```

Reply:
Yes! thank you. With this solution most of the comments bellow are fulfilled. I will send another patch shortly.